django_gcp/tasks/tasks.py

```diff
@@ -83,13 +83,23 @@ class Task(metaclass=TaskMeta):
     deduplicate = False
 
     def enqueue(self, **kwargs):
-        """Invoke a task (place it onto a queue for processing)"""
+        """Enqueues the given task for processing, unless 'self.manager.disable_execution' is True.
+        If `manager.disable_execution` is True, this method returns None without enqueuing the task.
```

Comment:
Think about this note as seen from the user's perspective. They'll have to investigate the source code to find out how to use it. Putting a more generic note, you could say something like "Some settings affect the behaviour of `enqueue()`, allowing bypass or immediate execution. See settings documentation for more." That'll also cover the fact that the eager execute option isn't documented.

Reply:
Edited docstring